media/js/devreg/slugify.js

```diff
@@ -27,9 +27,9 @@ function show_slug_edit(e) {
 }
 
 function slugify() {
-    if (z == null || z.unicode_letters) {
+    if (z === null || z.unicode_letters) {
         var slug = $('#id_slug');
-        url_customized = slug.attr('data-customized') == 0 ||
+        url_customized = slug.attr('data-customized') === 0 ||
```

Comment:
This might break. See this example for the background: http://jsbin.com/torivi/2/watch?html,js,console

I think we might want to swich to using `slug.data('customized') === 0` instead of attr.